app/config/battle_1vs1.py

```diff
@@ -0,0 +1,9 @@
+from app.config.knight import Knight
+
+
+def battle_1vs1(knight_1: str, knight_2: str) -> None:
+    pers = Knight.persons
```

Comment:
This variable is redundant better to do:
first_knight, second_knight = Knight.persons[knight_1], Knight.persons[knight_2]